keymap/keymap.go

```diff
@@ -42,6 +42,7 @@ const (
 	DocEnd  // end-of-doc Control / Alt / Shift +End
 	WordRight
 	WordLeft
+	// WordLeft is the final navigation function -- all above also allow Shift+ for selection.
 	FocusNext // Tab
```

Comment:
Add a line between the comment and next enum here to avoid erroneous association?